cypress/e2e/brewery/DatasetManager.cy.js

```diff
@@ -333,8 +333,6 @@ describe('Refresh dataset', () => {
         finalIngestionStatus: 'ERROR',
       };
       DatasetManager.switchToDatasetManagerView();
-      DatasetManager.getDatasetRefreshButton(DATASETS_TO_REFRESH[0].id).should('be.visible');
-      DatasetManager.getDatasetRefreshButton(DATASETS_TO_REFRESH[1].id).should('be.visible');
```

Comment:
Can we add some actions in the test to balance the removal of these lines?
For example, we could check that when we select a dataset in the list, the refresh button for this dataset is shown in the overview

Reply:
I think this is tested a bit further down where I added line 347 to select the dataset before clicking on its refresh button